catalog/utils.py

```diff
@@ -0,0 +1,167 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404, render
+
+from catalog.models import Category, Course
+
+
+def programTypeAndSlug(program) -> tuple:
```

Comment:
I'm not sure to understand why do you need to use a tuple with `aaaaaba`, `cap` and so one? It is not possible to just use the `program_type` in `buildOrderedProgramList()`?

Reply:
It was used to order the programs correctly but is now probably obsolete, we'll have to check what to merge and what to drop